lib/koans/15_processes.ex

```diff
@@ -17,9 +17,20 @@ defmodule Processes do
     assert is_pid(self()) == ___
   end
 
-  koan "New processes are spawned functions" do
+  koan "Like our current process, new processes are spawned functions" do
     pid = spawn(fn -> nil end)
 
+    assert is_pid(pid) == ___
+  end
+
+  koan "Processes exit after executing their function, unless they wait for something" do
```

Comment:
This might be clearer if the condition lead the statement.

This lesson still bugs me though. Ideally you'd want the learning process to be something like:
1. processes are spawned functions
2. processes die when the function exits
3. processes can send messages
4. a process waiting for a message will never die

Does that seem clearer? The trick is #2 is really hard. That's the issue we have with the current lesson, specifically that:

```
pid = spawn(fn -> nil end)
assert Process.alive?(pid)
```

should not pass intuitively, but it does because the `assert` executes in the current process before the spawned process exist. concurrency щ（ﾟДﾟщ）